File: Dart/1013.dart
import 'dart:io';

void main() {
  String inputs = stdin.readLineSync();
  List values = inputs.split(" ");
  List lst = values.map((e) => int.parse(e)).toList();
  int m = lst[0];

  for (int i = 0; i < lst.length; i++) {
    if (lst[i] > m) {
      m = lst[i];
    }
  }
  print(m.toString() + " eh o maior");
}

File: JavaScript/1013.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split(" ");

var a = parseInt(lines[0]);
var b = parseInt(lines[1]);
var c = parseInt(lines[2]);

var max = a;

if (b > max) max = b;
if (c > max) max = c;

console.log(max, "eh o maior");
